VimeoNetworking/Sources/Request+Cache.swift

```diff
@@ -32,12 +32,10 @@ public extension Request
     /// Generates a unique cache key for a request, taking into account endpoint and parameters
     var cacheKey: String
     {
-        var cacheKey = "cached" + self.path
+        let url = NSURL(string: self.path)
+        let urlPath = url?.path ?? ""
```

Comment:
@ghking - No, they are different and it's confusing because:
- `self.path` should only contain the `path` without the `URI`. However, because `self.path` is a `string` it contains the `path+uri`.
- `urlPath` is needed to extract the real path (= without the `URI`).